dev/cognitiveservices/data-plane/Language/analyzeconversations-authoring.json

```diff
@@ -1732,6 +2399,33 @@
       },
       "x-ms-client-name": "EvaluationSummary"
     },
+    "ConversationalAnalysisAuthoringExportedConversation": {
+      "description": "Represents a conversation.",
+      "required": [
+        "Location",
+        "summaryLocation"
+      ],
+      "type": "object",
+      "properties": {
+        "Location": {
```

Comment:
Why is `Location` capitalized? Properties should be camelCase as you've done throughout this PR.